src/earn/EarnActivePool.tsx

```diff
@@ -88,36 +102,51 @@ export default function EarnActivePool() {
               <TokenDisplay
                 amount={poolToken.balance}
                 showLocalAmount={true}
-                testID={`${networkConfig.arbUsdcTokenId}-totalAmount`}
-                tokenId={networkConfig.arbUsdcTokenId}
+                testID={`${depositTokenId}-totalAmount`}
+                tokenId={depositTokenId}
               />
             )}
           </View>
         </View>
-        <View style={styles.buttonContainer}>
-          <Button
-            onPress={() => {
-              // TODO (act-1180) create earn review withdraw screen
-              // Will navigate to this screen with appropriate props
-              // fire analytics
-            }}
-            text={t('earnFlow.activePools.exitPool')}
-            type={BtnTypes.SECONDARY}
-            size={BtnSizes.FULL}
-            style={styles.button}
-          />
-          <Button
-            onPress={() => {
-              // TODO (act-1176) create earn enter amount screen
-              // Will navigate to this screen with appropriate props
-              // fire analytics
-            }}
-            text={t('earnFlow.activePools.depositMore')}
-            type={BtnTypes.PRIMARY}
-            size={BtnSizes.FULL}
-            style={styles.button}
-          />
-        </View>
+        {cta === 'ExitAndDeposit' && (
+          <View style={styles.buttonContainer}>
+            <Button
+              onPress={() => {
+                // TODO (act-1180) create earn review withdraw screen
+                // Will navigate to this screen with appropriate props
+                // fire analytics
+              }}
+              text={t('earnFlow.activePools.exitPool')}
+              type={BtnTypes.SECONDARY}
+              size={BtnSizes.FULL}
+              style={styles.button}
+            />
+            <Button
+              onPress={() => {
+                // TODO (act-1176) create earn enter amount screen
+                // Will navigate to this screen with appropriate props
+                // fire analytics
+              }}
+              text={t('earnFlow.activePools.depositMore')}
+              type={BtnTypes.PRIMARY}
+              size={BtnSizes.FULL}
+              style={styles.button}
+            />
+          </View>
+        )}
+        {cta === 'ViewPools' && (
+          <View style={styles.buttonContainer}>
+            <Button
+              onPress={() => {
+                navigate(Screens.TabDiscover)
+              }}
+              text={t('earnFlow.activePools.viewPools')}
+              type={BtnTypes.SECONDARY}
```

Comment:
same as above

Reply:
Fixed in 483b32de2aa2a3137edd2d230f1d02281e196c9a!